codalab/apps/web/tests/test_submission_download.py

```diff
@@ -60,16 +65,19 @@ def setUp(self):
     def test_submission_info_download_when_submitter_returns_200(self):
         self.client.login(username="participant", password="pass")
         resp = self.client.get(self.url)
-        # TODO: This seems to have previously returned a 200, but now returns a 302. It is a redirect to AWS
-        self.assertEqual(resp.status_code, 302)
+        if settings.USE_AWS:
```

Comment:
Why don't you set this once with a variable like "expected AWS return status code"

Reply:
That probably makes more sense, but for tests with file contents we'll still need a conditional for whether it's a redirect and we need to hit AWS for the file contents. If that's preferred I'll push the changes.